public/resources/default-files/html.txt

```diff
@@ -1,15 +1,13 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Made with Thimble</title>
-</head>
-<body>
-
-  <h1>Welcome to Thimble</h1>
-
-  <p>Make something <strong>amazing</strong> with the web!</p>
-
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <title>Made with Thimble</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
```

Comment:
Is this the final version we're rolling with?

![image](https://cloud.githubusercontent.com/assets/25212/22315027/0f88867a-e319-11e6-8029-923eaee182c3.png)